cmd/mothership/mothership/start/auditlog.go

```diff
@@ -107,8 +131,50 @@ func auditLogRequest(w http.ResponseWriter, r *http.Request, l *zap.Logger) {
 			return
 		}
 		r.Body = ioutil.NopCloser(bytes.NewBuffer(reqBody))
-		l = l.With(zap.String("requestBody", string(reqBody)))
+		logData.RequestBody = string(reqBody)
+	}
+
+	if ip, ok := os.LookupEnv("POD_IP"); ok {
+		logData.IP = ip
```

Comment:
I guess the IP should be the source IP of the incoming request as this is relevant info for the auditing